django/forms/renderers/templates.py

```diff
@@ -0,0 +1,61 @@
+import os
```

Comment:
I think at one point I had it split out into `renderers/django.py` and `renderers/jinja2.py`. I think it makes sense to just be `renderers.py`.